config/environments/production.rb

```diff
@@ -53,16 +47,19 @@
   config.log_level = :debug
 
   # Prepend all log lines with the following tags.
-  # config.log_tags = [ :subdomain, :uuid ]
-
-  # Use a different logger for distributed setups.
-  # config.logger = ActiveSupport::TaggedLogging.new(SyslogLogger.new)
+  # config.log_tags = [ :request_id ]
```

Comment:
I'd rather that either we delete the commentary entirely to leave just the config, or we leave the default Rails commentary.  Having custom commented out commentary seems a little pointless.